SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -313,6 +313,37 @@ ESchemaComponentType USpatialClassInfoManager::GetCategoryByComponentId(Worker_C
 	return ESchemaComponentType::SCHEMA_Invalid;
 }
 
+const FRPCInfo* USpatialClassInfoManager::GetRPCInfo(UObject* Object, UFunction* Function)
+{
+	check(Object && Function);
+
+	if (Object == nullptr ||
```

Comment:
whoa lol, you have a check above to make this not happen. Does this if need to exist?

Reply:
Can change it back and make the function return a const ref otherwise?

Reply:
Changed return value to constref.